internal/rbd/controllerserver.go

```diff
@@ -90,7 +90,7 @@ func (cs *ControllerServer) validateVolumeReq(ctx context.Context, req *csi.Crea
 	return nil
 }
 
-func (cs *ControllerServer) parseVolCreateRequest(ctx context.Context, req *csi.CreateVolumeRequest) (*rbdVolume, error) {
+func (cs *ControllerServer) parseVolCreateRequest(ctx context.Context, req *csi.CreateVolumeRequest) (*rbdVolume, bool, error) {
```

Comment:
Please document the `bool` for return value. Users of the function do not know what the bool means.

Reply:
The callers of this function need to be adjusted in the same commit. This is currently not compiling, which means `git bisect` will break.